ecc/bls12381/g1.go

```diff
@@ -174,10 +182,40 @@ func (g *G1) Add(P, Q *G1) {
 // ScalarMult calculates g = kP.
 func (g *G1) ScalarMult(k *Scalar, P *G1) { g.scalarMult(k.Bytes(), P) }
 
+// ScalarMultVarTime calculates g=kP, leaking k, for short k
+func (g *G1) ScalarMultVarTimeShort(k *Scalar, P *G1) { g.scalarMultVarTimeShort(k.Bytes(), P) }
```

Comment:
`ScalarMultVarTimeShort`--> `ScalarMultVarTime`

Not sure if we want to expose this function

Reply:
let's keep it internal. I added short to emphasize that the tradeoffs are different.